### DISCUSSION TITLE
Is there a way to use dictionnary of enum and string with FluentCombobox

### DISCUSSION THREAD

**Discussion by jobou363** [microsoft/fluentui-blazor#594] [Q&A]
I'm using fluentui blazor 3.0.0-RC.1.

I'm wondering if there is an option or a way i don't see to configure FluentCombobox with a key pair int-string.

I'm use to fill my combobox using enum value (int) and description but fluent option require (string - string) or (int - int).

I would have needed (int - string) or (enum - string).

`
<FluentCombobox @bind-Value=SelectedProvince TOption="string" >
          @foreach (var province in Provinces)
          {                            
              <FluentOption Value=@province.Key >@province.Value</FluentOption>
          }
</FluentCombobox>

@code{
  public Dictionary<Province, string> Provinces = new Dictionary<Province, string>();
  public Province SelectedProvince = 0;

  protected override async Task OnInitializedAsync()
  {        
             Provinces = Enum.GetValues(typeof(Province)).Cast<Province>().ToDictionary(pro => ((int)pro).ToString(), pro => 
                               Enumeration.GetDescriptionLocaliser(pro,CultureInfo.CurrentCulture));

  }
}
`

**Comment by vnbaaij** [comment]
This is, by definition, not possible. A Combobox allows for typing and thereby 'selecting' an option that is not part of the initial list. There is no way then to the specify what the value-part of that typed option would be.
